Adyen/client.py

```diff
@@ -452,6 +461,7 @@ def call_checkout_api(self, request_data, action, idempotency_key=None,
                 https://docs.adyen.com/developers/checkout/api-integration
             service (str): This is the API service to be called.
             action (str): The specific action of the API service to be called
+            path_param (str): This is used to pass the id or referenceID to the API sercie
```

Comment:
```suggestion
            path_param (str): This is used to pass the ID or referenceID to the API service
```

Reply:
Will fix this and open a new PR to develop,